### PR TITLE
fast fixture setup: parallel endpoint registration

### PR DESCRIPTION
**Pull request by hackaugusto**
[ci integration]

- added method start_async in RaidenService
- moved the call to the end point registry from the constructor to the
start method
- moved the call to `start` out of the constructor
- moved the handling of the filelock.Timeout exception to the CLI layer
- removed the exception handler for failed end point registration. The
default exception handling mode is to exit the node, so the `sys.exit`
is only forbidden
- removed outdated speed_simulation

### CODE REVIEW

File: raiden/ui/cli.py [hunk 1]
```diff
@@ -689,6 +690,15 @@ def app(
         discovery,
     )
 
+    try:
+        raiden_app.start()
+    except filelock.Timeout:
+        print(
+            f'FATAL: Another Raiden instance already running for account {address_hex} on '
+            f'network id {network_id}',
```
Comment (ulope) [inline raiden/ui/cli.py]: Maybe use `constants.ID_TO_NETWORKNAME` to translate the network id and `to_checksum_address` for the eth address to have a nicer message.

**Comment by ulope** [comment]
Doesn't `raiden/tests/utils/network.py::create_apps()` also need `app.start()`?

**Comment by hackaugusto** [comment]
> Doesn't raiden/tests/utils/network.py::create_apps() also need app.start()?

It's done by the functions that are calling `create_apps`, [here](https://github.com/raiden-network/raiden/pull/1743/files#diff-debebe59b75c7f8628aff99357e5ef8dR154) and [here](https://github.com/raiden-network/raiden/pull/1743/files#diff-debebe59b75c7f8628aff99357e5ef8dR71)

**Comment by ulope** [comment]
Depends, if you think it's useful to be able to call start later when using that fixture then ok. Might one more thing to forget though...

**Comment by LefterisJP** [comment]
LGTM. Will make a commit to run integration and remove it later.

**Pull request merged**